pkg/util/util.go

```diff
@@ -45,6 +47,71 @@ func GenerateRandomString(n int) string {
 	return string(b)
 }
 
+// StringInSlice returns if a passed string exists in the passed slice of strings
+func StringInSlice(checkStr string, strSlice []string) bool {
```

Comment:
We won't need this if we implement the way discussed in https://github.com/redhat-developer/odo/pull/741/files#r219470863

Reply:
Same as in https://github.com/redhat-developer/odo/pull/741/files#r219478594 and https://github.com/redhat-developer/odo/pull/741/files#r219478808

Reply:
we don't need this function any more. It's not used anywhere except the test for the function.